uitools/toolkitwidgets/docs/AuthenticationView.md

```diff
@@ -0,0 +1,18 @@
+[![ArcGIS Maps SDK for Qt](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt-0b5394)](https://developers.arcgis.com/qt/) [![ArcGIS Maps SDK for Qt toolkit](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt%20toolkit-ea4d13)](https://github.com/Esri/arcgis-maps-sdk-toolkit-qt) [![Qt Widgets UI components](https://img.shields.io/badge/Qt%20Qt%20Widgets%20UI%20components-ea4d13)](../../toolkitwidgets/)
+
+# AuthenticationView - Qt Widgets UI control/Class
```

Comment:
Not sure what "control/class" refers to. I think you could probably just leave it at "Qt Widgets UI"

Reply:
Fixed with commit: https://github.com/Esri/arcgis-maps-sdk-toolkit-qt/pull/660/commits/94d39feb83d333ec756547ac9cf37d3ee93bdec7